Source/WebKit/UIProcess/API/Cocoa/_WKInspectorWindow.h

```diff
@@ -27,11 +27,18 @@
 
 #if !TARGET_OS_IPHONE
 
+NS_ASSUME_NONNULL_BEGIN
+
+@class WKWebView;
+
 WK_CLASS_AVAILABLE(macos(10.15))
 @interface _WKInspectorWindow : NSWindow
 
-@property (nonatomic, getter=isForRemoteTarget) BOOL forRemoteTarget;
+@property (nonatomic, readonly, getter=isForRemoteTarget) BOOL forRemoteTarget;
+@property (nonatomic, nullable, readonly, weak) WKWebView *inspectedWebView;
```

Comment:
This new property should probably get availability annotations for TBA versions. e.g. `@property (nonatomic, nullable, readonly, weak) WKWebView *inspectedWebView WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))`

Reply:
I think I would just add `WK_API_AVAILABLE(macos(WK_MAC_TBA))`. This class is excluded on everything but non-Catalyst Mac (since NSWindow isn't available), so the iOS availability isn't particularly meaningful.

Reply:
Oh good point, yeah this is Mac only, so no need for iOS availability 😅